content/articles/object-detection-with-yolov5-and-pytorch/index.md

```diff
@@ -0,0 +1,197 @@
+﻿
+### Object Detection with YOLOV5 and Pytorch
+
+Object detection is usually one of the most interesting computer vision tasks a beginner or an enthusiast can use to jump-start their career in computer vision and artificial Intelligence.
+
+In this article, we will look at the basic knowledge on object detection before proceeding to perform object detection using [YOLOv5](https://github.com/ultralytics/yolov5) and the [COCO dataset](https://cocodataset.org/#home).
+
+### Introduction
+
+**Object detection** is a computer vision task that involves identifying instances of an object of a certain class within an image or a video.
+With a wide are of applications like self-driving cars, security, manufacturing, etc. ,Object detection is a very interesting computer vision task to get started with.
+
+Object detection has two main state-of-art methods namely:
+
+1.  One-stage methods - these are mainly focused on the inference speed. Examples include: YOLO, RetinaNet, and SSD
+2.  Two-stage methods - these are mainly focused on detection accuracy. Examples include Mask R-CNN, Faster R-CNN, and Cascade R-CNN.
+
+To check our model’s accuracy, we will be using the popular object detector metric _Average Precision_.
+
+In this tutorial will be using  **YOLO** (_You Only Look Once_) specifically [YOLOv5](https://github.com/ultralytics/yolov5) which focuses on inference speed and accuracy.
+
+Slowly becoming the industry’s standard for object detection, YOLO is a fast and accurate approach to object detection. The algorithm only looks at an image once and detects  what objects are present and their location.
+
+YOLO works by splitting images into a grid where each grid cell identifies an object by itself. The grid cells predict all the bounding boxes and gives each of them a confidence scores to detemine the accuracy of each prediction.
+
+When it comes to perfomance, YOLO outperforms other object detectors by far. In real-time, it is able to process images at a rate of around 155 frames per second, achieving double the mAP(Mean Average Precision) of other object detectors like R-CNN.
+
+YOLOv5, the latest release of the YOLO family is a group of compound-scaled object detection models trained on the COCO dataset used for model ensembling, Test Time Augmentation (TTA) and hyperparameter evolution.
```

Comment:
`model ensembling, Test Time Augmentation (TTA) and hyperparameter evolution.`

Add single line explanation about these